frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/date_cell/date_editor.dart

```diff
@@ -111,12 +111,14 @@ class _CellCalendarWidgetState extends State<_CellCalendarWidget> {
       child: BlocBuilder<DateCellCalendarBloc, DateCellCalendarState>(
         buildWhen: (p, c) => p != c,
         builder: (context, state) {
+          bool includeTime = state.dateCellData
```

Comment:
Style suggestion:
place a comma after dateData.includeTime

Reply:
> place a comma after dateData.includeTime

Sure thing

> Would it be better if creating a property in the state called includeTime.

Should we consider just unpacking the entire `DateCellData` into the state and remove `dateCellData`? `time` is already there as well.

![image](https://user-images.githubusercontent.com/71320345/223935043-4911a5c0-2c2c-4427-91e6-bc9da1b844e9.png)